define/define.py

```diff
@@ -0,0 +1,45 @@
+import urllib2
+import json
+import random
+from datamuse.datamuse import Datamuse
+
+def get_rhyme(word):
+    request = Datamuse.get_perfect_rhymes(word)
+    results = json.loads(request.read().decode("utf-8"))
+    return str(random.choice(results)['word'])
+
+def define_word(word):
+    request = urllib2.urlopen('http://api.pearson.com/v2/dictionaries/ldoce5/entries?headword=' + word)
+    results = json.loads(request.read().decode("utf-8"))
+    entry = random.choice(results['results'])
+    definition = random.choice(entry['senses'][0]['definition'])
+    return str(definition)
+
+def define_command(args):
+    word = args.split()[0]
+
+    rhyme_word = get_rhyme(word)
+    while rhyme_word == word:
```

Comment:
Please make this a for loop (max of 5 or 10 tries) in case there's only one result for whatever reason.